worlds/Arena.wbt

```diff
@@ -280,43 +280,149 @@ Solid {
   name "South Wall"
 }
 Solid {
-  translation -2.9 0.15 -0.333
-  rotation 0.5773499358549823 -0.5773489358550934 0.5773519358547601 2.09439
+  # An issue with the DistanceSensor in webots-2020b-r1 
+  # causes certain faces to not be detected
+  # It appears that the issue does not occur in IndexedFaceSet
+  # when all faces have a common vertex with all others
   children [
-    DEF SHAPE Shape {
-      appearance PBRAppearance {
-        baseColor 0.095395215 0.22841774 0.8000001
-        roughness 1
-        metalness 0
-      }
-      geometry TexturedParallelepiped {
-        size 0.28 4.89 0.3
-        angles -1.085 0 0
-      }
+    Solid {
+      children [
+        DEF SHAPE Shape {
+          appearance PBRAppearance {
+            baseColor 0.095395215 0.22841774 0.8000001
+            roughness 1
+            metalness 0
+          }
+          geometry IndexedFaceSet {
```

Comment:
Presumably it's not possible to have an `IndexedFaceSet` which declares all 6 exterior faces in one go?